src/paths.rs

```diff
@@ -45,6 +45,21 @@ pub struct PathItem {
 }
 
 impl PathItem {
+    pub fn into_iter(self) -> impl Iterator<Item = Operation> {
```

Comment:
Can you please add a doc comment?

Reply:
Thanks for adding the comment. Sorry to yank you around on this, but two thoughts:

1. Should this instead be an `impl IntoIterator`? I think this allows it to be used with for loops
2. I think I made a mistake with `iter()` in that the Item type should have been `(&'static str, Operation)`. I've found that many consumers need that operation-type information. What do you think?